ocs_ci/ocs/cluster.py

```
@@ -2407,14 +2407,22 @@ def is_ec_pool_supported():
         return False
     if not is_lso_cluster():
         return False
-    if get_failure_domin() != "host":
-        return False
-    if len(pod.get_osd_pods()) < 3:
+    try:
+        if get_failure_domin() != "host":
+            return False
+        if len(pod.get_osd_pods()) < 3:
+            return False
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)